ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -30,12 +33,14 @@ public enum PillButtonStyle: Int {
             return UIColor(light: Colors.primary(for: window), dark: Colors.surfaceQuaternary)
         case .filled:
             return UIColor(light: Colors.surfacePrimary, dark: Colors.surfaceQuaternary)
+        case .transparent:
+            return Colors.primaryTint40(for: window)
```

Comment:
Looking at the xcode declaration, I see the warning
```
'primaryTint40' is deprecated: renamed to 'setProvider(_:forWindow:)'
Use 'setProvider(_:forWindow:)' instead
```

Should we not be using this API anymore and replace it with the ColorProviding protocol @anandrajeswaran added?

Reply:
this method is using the color providing protocol. Its implementation has a warning because we still need to support clients that do not use the protocol, so it calls deprecated var primaryTint40 in that case

---
In reply to: [447136201](https://github.com/microsoft/fluentui-apple/pull/105#discussion_r447136201) [](ancestors = 447136201)